src/main/scala/format/Forsyth.scala

```diff
@@ -159,7 +159,7 @@ object Forsyth {
       exportBoard(game.board) + exportCrazyPocket(game.board),
       game.player.letter,
       exportCastles(game.board),
-      game.situation.enPassantSquare.map(_.toString).getOrElse("-"),
+      "-",
```

Comment:
Unfortunately we need to keep this for backwards compability.

Reply:
Maybe we should just delete the games that are corrupted by this pawn bug?